src/upgrade-to-1_7.ts

```diff
@@ -0,0 +1,133 @@
+import { rmdirSync, writeFileSync } from 'fs'
```

Comment:
The idea behind this was to help countries migrate their roles automatically to a `roles.json` file, without us having to write a comprehensive documentation on how to do it. The ideal time for running the script would probably be after they run `git pull countryconfig master` and get merge conflicts. Ideas welcome..

Reply:
Using husky and a `pre-merge-commit` or `post-merge` hook?